pages/common/mvn.md

```diff
@@ -0,0 +1,23 @@
+# mvn
+
+> Apache Maven. A build automation tool used for JVM environment projects.
```

Comment:
Please move the 2nd sentence to a new line.

Reply:
The maven docs describe mvn as:

"Tool for building and managing Java-based projects."